### PR TITLE
Grammatically updated the vision tutorial

### PR DESCRIPTION
**Pull request by 0tist**
grammatically updated, corrected typos and improved expression for better understanding

### CODE REVIEW

**Comment by 0tist** [comment]
Hi @hamelsmu could you please review the PR

File: nbs/23_tutorial.vision.ipynb [hunk 1]
```diff
@@ -495,7 +494,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This time we resized to a larger size before batching, and we add `batch_tfms`. `aug_transforms` is a function that provides a collection of data augmentation transforms with defaults we found worked very well on most datasets (you can customize each one by passing the right arguments)."
+    "This time we resized to a larger size before batching, and we add `batch_tfms`. `aug_transforms` is a function that provides a collection of data augmentation transforms with defaults that are observed to perform well on most datasets.`aug_trsnforms` can be cutomized to suit one's need."
```
Comment (hamelsmu) [inline nbs/23_tutorial.vision.ipynb:497]:
```suggestion
    "This time we resized to a larger size before batching, and we added `batch_tfms`. `aug_transforms` is a function that provides a collection of data augmentation transforms with defaults we found that perform well on many datasets.  You can customize these transforms by passing appropriate arguments to `aug_transnforms`."
```

Comment (hamelsmu) [inline nbs/23_tutorial.vision.ipynb:497]:
```suggestion
"This time we resized to a larger size before batching, and we added `batch_tfms`. `aug_transforms` is a function that provides a collection of data augmentation transforms with defaults we found that perform well on many datasets.  You can customize these transforms by passing appropriate arguments to `aug_transnforms`."
```

Comment (hamelsmu) [inline nbs/23_tutorial.vision.ipynb:497]:
```suggestion
    "This time we resized to a larger size before batching, and we added `batch_tfms`. `aug_transforms` is a function that provides a collection of data augmentation transforms with defaults we found that perform well on many datasets.  You can customize these transforms by passing appropriate arguments to `aug_transnforms`."
```

Comment (hamelsmu) [reply nbs/23_tutorial.vision.ipynb:497]: Don't think you should change the original meaning of this sentence?  I'll make some small modifications to revert parts of it back

**Comment by hamelsmu** [comment]
going to merge this via #2742

**Pull request closed without merging**